### PR TITLE
Sleeves and Gang Clash UI Refresh

### PR DESCRIPTION
**Pull request by nickofolas**
This PR aims to refresh the sleeves UI and gang clash UI similarly to the recent gang UI update. In particular, it incorporates similar design motifs (i.e. the skill colors matching the character overview).

Todo:
- [x] Main sleeves tab
- [x] Sleeves augmentation modal
- [ ] ~~Re-sleeve page~~ (will remain untouched due to talks of replacing re-sleeving entirely)
- [x] Gang clash UI

### Screenshots

**Sleeves**
image
image

**Gang Clash**
The previous wall of text has been hidden behind a modal to allow quicker access to relevant information.

Furthermore, said information is now more condensed, but also visually separated similarly to the "Create Programs" UI.

image
image

### Tests
This PR is mainly a UI change, thus, there are no concerns to be had with API breakage.

As far as UI issues are concerned, I've fixed any I find in my testing of the changes.

### CODE REVIEW

File: src/PersonObjects/Sleeve/ui/StatsElement.tsx [hunk 1]
```diff
@@ -1,31 +1,110 @@
...
+  const classes = useStyles();
+
+  return (
+    <Table sx={{ display: 'table', mb: 1, width: '100%' }}>
+      <TableBody>
+        <StatsRow name="City" color={Settings.theme.primary} data={{ content: props.sleeve.city }} />
+        <StatsRow name="HP" color={Settings.theme.hp}
+          data={{ content: `${numeralWrapper.formatHp(props.sleeve.hp)} / ${numeralWrapper.formatHp(props.sleeve.max_hp)}` }}
+        />
+        <StatsRow name="Hacking" color={Settings.theme.hack} data={{ level: props.sleeve.hacking, exp: props.sleeve.hacking_exp }} />
```
Comment (phyzical) [inline src/PersonObjects/Sleeve/ui/StatsElement.tsx:37]: i wonder if this stuff can be abstracted i feel like ive seen these a few times

Comment (phyzical) [inline src/PersonObjects/Sleeve/ui/StatsElement.tsx:37]: this could be abstracted out think i saw these higher up in the pr already

**Review by phyzical — APPROVED** [review]
LGTM, not i just scanned the pr didnt run it 

Looking swanky 👍

File: src/PersonObjects/Sleeve/ui/StatsElement.tsx (hunk 1 above)
Comment (nickofolas) [reply src/PersonObjects/Sleeve/ui/StatsElement.tsx:37]: how so? the `<StatsRow>` component is already an abstraction itself heh

**Pull request merged**